website/content/en/docs/deployment/vanilla/guide-terraform.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Terraform Deployment Guide"
+description = "Deploy the vanilla distribution of Kubeflow on AWS using Terraform"
+weight = 30
++++
+
+> Note: Terraform deployment options are still in preview.
+
+## Prerequisites
+
+Be sure that you have satisfied the [installation prerequisites]({{< ref "../prerequisites.md" >}}) before working through this guide.
+
+Specifially, you must:
+- [Create a Ubuntu environment]({{< ref "../prerequisites/#create-ubuntu-environment" >}})
+- [Clone the repository]({{< ref "../prerequisites/#clone-repository" >}})
+- [Install the necessary tools]({{< ref "../prerequisites/#create-ubuntu-environment" >}})
+
+## Deployment Steps
```

Comment:
- Needs some intro which explains what the deployment will do for e.g. creates a VPC, EKS cluster, installs all Kubeflow components

- Needs a section called customization which explains where are the defaults set and how to change things

@rrrkharse